pkg/virt-handler/vm.go

```diff
@@ -756,14 +756,16 @@ func (d *VirtualMachineController) defaultExecute(key string,
 	// set true to ensure that no updates to the current VirtualMachineInstance state will occur
 	forceIgnoreSync := false
 
-	log.Log.V(3).Infof("Processing vmi %v, existing: %v\n", vmi.Name, vmiExists)
+	log.Log.Infof("Processing event %v", key)
 	if vmiExists {
-		log.Log.V(3).Infof("vmi is in phase: %v\n", vmi.Status.Phase)
+		log.Log.Infof("Vmi %s in phase %v", vmi.Name, vmi.Status.Phase)
+	} else {
+		log.Log.Info("Vmi does not exist")
 	}
-
-	log.Log.V(3).Infof("Domain: existing: %v\n", domainExists)
```

Comment:
I wonder why those logs used to be on V(3), they shouldn't be written too often, right?

Reply:
Should not happen more often that the log `Received Domain Event of type MODIFIED` we are already logging and I think they'll add a lot of useful information.